experimenter/tests/integration/nimbus/pages/demo_app/frontend.py

```diff
@@ -14,7 +14,7 @@ def wait_for_result_text(self, text_list):
 
         return self.wait_for_and_find_element(By.XPATH, xpath, description=None)
 
-    def fill_and_send_form_data(self, client_id, context):
+    def fill_and_send_form_data(self, client_id, context, nimbus_preview=False):
```

Comment:
We have this DemoAppPage class here, can we move all the string constants onto constants here with some convenience methods for interacting with this page so the tests aren't quite so long? 🙏